resolwe_bio/processes/expression/htseq_count.yml

```diff
@@ -323,3 +355,6 @@
       re-save-file htseq_output "${NAME}_HTseq_rc.txt.gz"
       re-save exp_type 'Read count'
       re-save source {{gff.source}}
+      re-save species {{alignments.species}}
+      re-save build {{alignments.build}}
+      re-save feature_type {{feature_type}}
```

Comment:
This will fail, since there is no input with name `feature_type`.

Reply:
Fixed, thank you!